contracts/NFTs/Base/NftBaseUpgradeable.sol

```diff
@@ -21,6 +21,7 @@ abstract contract NftBaseUpgradeable is
     using CountersUpgradeable for CountersUpgradeable.Counter;
 
     CountersUpgradeable.Counter private _tokenIdCounter;
+    bytes32 public constant ADMIN_ROLE = keccak256("ADMIN_ROLE");
```

Comment:
We don't import this from `Roles.sol`? Why?

Reply:
I use the `Roles.sol` only for the `DIMORegistry`. The NFTs are kind of the part.